vhd/packer/sysprep.ps1

```diff
@@ -0,0 +1,38 @@
+# Stop and remove Azure Agents to enable use in Azure Stack
+# If deployed in azure vm the agents will be added to the vms at deployment time
+Stop-Service WindowsAzureGuestAgent
+Stop-Service WindowsAzureNetAgentSvc
+Stop-Service RdAgent
+# Stop-Service WindowsAzureTelemetryService
```

Comment:
Why not stop and delete WindowsAzureTelemetryService?

Reply:
The WindowsAzureTelemetryService has been merged into the RD agent in latest version of the agents: https://docs.microsoft.com/en-us/azure/virtual-machines/troubleshooting/windows-azure-guest-agent

> Starting in version 2.7.41491.971 of the VM Guest Agent, the Telemetry component is included in the RDAgent service, Therefore, you might not see this Telemetry service listed in newly created VMs.

This was left over from earlier refactoring.  I'll remove it from the file